hcipy/aperture/realistic.py

```diff
@@ -2,8 +2,63 @@
 from ..field import make_hexagonal_grid, Field
 from .generic import make_spider, circular_aperture, hexagonal_aperture, make_segmented_aperture, make_spider_infinite, make_obstructed_circular_aperture
 
-def make_vlt_aperture():
-	pass
+def make_vlt_aperture(normalized=False, with_spiders=True, orientation=0):
+	'''Make the VLT aperture.
+
```

Comment:
Can you add a source for the values used in this function? Citation or even webpage is fine, but I'd like to have at least some source listed here.